### PR TITLE
Develop acao

### PR DESCRIPTION
**Pull request by alan-cao**
Fix 29683926 - CREATE-DOMAIN-SCRIPT.SH HARD-CODE CREATEFMWDOMAIN.PY PATH IN JRF DOMAIN CREATION

### CODE REVIEW

**Comment by xiancao** [comment]
do you need to merge it to release/2.2 too?

**Comment by alan-cao** [comment]
> do you need to merge it to release/2.2 too?

File: kubernetes/samples/scripts/create-fmw-infrastructure-domain/wlst/create-domain-script.sh [hunk 1]
```diff
@@ -22,4 +22,4 @@ wlst.sh -skipWLSModuleScanning \
         -managedServerPort ${CUSTOM_MANAGEDSERVER_PORT} \
         -prodMode ${CUSTOM_PRODUCTION_MODE} \
         -managedServerCount ${CUSTOM_MANAGED_SERVER_COUNT} \
-        -clusterName ${CUSTOM_CLUSTER_NAME}
\ No newline at end of file
+        -clusterName ${CUSTOM_CLUSTER_NAME}
```
Comment (ddsharpe) [inline kubernetes/samples/scripts/create-fmw-infrastructure-domain/wlst/create-domain-script.sh]: Why would you eliminate the last newline here?  Normally, it is best practice to end text and script files with newline.

Comment (alan-cao) [reply kubernetes/samples/scripts/create-fmw-infrastructure-domain/wlst/create-domain-script.sh]: The original file doesn't end with a new line. I don't know why it shows a diff. I didn't change to that line.

Comment (alan-cao) [reply kubernetes/samples/scripts/create-fmw-infrastructure-domain/wlst/create-domain-script.sh]: Anyway, I added a newline in the end to follow the best practice. Please review.

**Pull request merged**